app/views/foreman/job_templates/install_errata_-_katello_ansible_default.erb

```diff
@@ -15,12 +15,10 @@ kind: job_template
 %>
 
 <% if @host.operatingsystem.family == 'Suse' -%>
-<% advisories = input(:errata).split(',').join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisories}") -%>
-<% elsif @host.operatingsystem.family == 'Debian' -%>
-<% advisories = input(:errata).split(',').map { |e| @host.debs_for_erratum(e) }.join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "apt-get -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" -y --only-upgrade install -y #{advisories}") %>
+  <% advisories = input(:errata).split(',').join(' ') -%>
+  <% render_error(N_('At least one errata ID is required')) if advisories.blank? -%>
```

Comment:
I don't think we will ever hit this code path, since [this](https://github.com/Katello/katello/pull/10444#discussion_r1124603955) happens before the template rendering is attempted. Thoughts on removing these two?